daprdocs/content/en/dotnet-sdk-docs/dotnet-development/_index.md

```diff
@@ -0,0 +1,181 @@
+---
+type: docs
+title: "Dapr .NET SDK Development"
+linkTitle: "Development"
+weight: 1000
+description: Learn about local development options for .NET Dapr applications
+no_list: true
```

Comment:
If `Development` will eventually be a directory with many pages under it, let's leave the PR as is, with it in it's own folder and filename `_index.md`. If it's just a single page of content, let's move it up a directory and rename it to `dotnet-development.md`.